tests/ui/test_home.py

```diff
@@ -3,15 +3,19 @@
 from pages.desktop.home import Home
 
 
+@pytest.mark.django_db
 @pytest.mark.nondestructive
-def test_there_are_ten_most_popular_extensions(base_url, selenium):
+def test_there_are_ten_most_popular_extensions(
+        base_url, selenium, live_server, initial_data):
```

Comment:
As `live_server` is function scoped, running tests in parallel will cause multiple servers to start, and each test would need the unique URL from each server.